packages/babel-polyfill/package.json

```diff
@@ -8,7 +8,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-polyfill",
   "main": "lib/index.js",
   "dependencies": {
-    "core-js": "^1.0.1",
+    "core-js": "^2.0.0",
```

Comment:
It makes sense. But now these versions bound to `babel-core` - it can create additional misunderstanding to users. How to solve this problem?